### PR TITLE
U4X-740: Adding an edit feature to the current regimen on the care panel

### PR DESCRIPTION
**Pull request by Daphne210**
https://github.com/user-attachments/assets/f52dcfd6-0a79-4ba7-a455-37995e277851

https://metsprogramme.atlassian.net/browse/U4X-740

### CODE REVIEW

**Comment by slubwama** [comment]
@akileng56  this requires your extra review.

**Comment by slubwama** [comment]
@akileng56  @jabahum help review this and get it in if it passes.

**Comment by akileng56** [comment]
@Daphne210 this is supposed to edit the regimen but when u open the form, regimen field is not editable

File: packages/esm-care-panel-app/src/regimen-editor/regimen-button.component.tsx [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-    >
-      {t('editRegimen', 'Edit')}
-    </Link>
+    <>
+      <Link
+        className={styles.linkName}
+        onClick={() =>
+          launchPatientWorkspace('patient-form-entry-workspace', {
+            formInfo: {
+              encounterUuid: '',
```
Comment (akileng56) [inline packages/esm-care-panel-app/src/regimen-editor/regimen-button.component.tsx:22]: Is this a required field? if not remove it

File: packages/esm-care-panel-app/src/program-summary/program-summary.component.tsx [hunk 2]
```diff
@@ -147,12 +147,12 @@ const ProgramSummary: React.FC<ProgramSummaryProps> = ({ patientUuid, programNam
                           ? data?.TB?.lastTbEncounter?.regimenShortDisplay
                           : t('neverOnTbRegimen', 'Never on TB regimen')}
                       </span>
-                      <RegimenButton
+                      {/* <RegimenButton
```
Comment (akileng56) [inline packages/esm-care-panel-app/src/program-summary/program-summary.component.tsx:150]: Remove code that is not in-use

**Comment by Daphne210** [comment]
> @Daphne210 this is supposed to edit the regimen but when u open the form, regimen field is not editable

The clinical implementation of changing a regimen is only through regimen switch/substitution but in plain English it's editing

File: packages/esm-care-panel-app/src/program-summary/program-summary.component.tsx (hunk 2 above)
Comment (Daphne210) [reply packages/esm-care-panel-app/src/program-summary/program-summary.component.tsx:150]: This has been removed

File: packages/esm-care-panel-app/src/regimen-editor/regimen-button.component.tsx (hunk 1 above)
Comment (Daphne210) [reply packages/esm-care-panel-app/src/regimen-editor/regimen-button.component.tsx:22]: No it is not, it has been removed

**Comment by Daphne210** [comment]
>
Screenshot (843)
 @Daphne210 this is supposed to edit the regimen but when u open the form, regimen field is not editable

Have changed the word to "Change Regimen" don't know if that's better or worse

**Review by akileng56 — APPROVED** [review]
